integration/js/README.md

```diff
@@ -2,13 +2,96 @@
 
 This guide will help you setup and run integration tests for [Chime SDK](https://github.com/aws/amazon-chime-sdk-js) using [Chime SDK Browser Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/browser).
 
+## Testing locally
+
+To run integration tests you will need:
+
+- Node 10 or higher. Node 14 is supported.
+- npm 6.11 or higher. 6.14.8 is supported.
+- [KITE](https://github.com/webrtc/KITE).
+
+### Installing KITE on macOS
```

Comment:
This is common for both locally and SauceLabs right. Could we move it outside to before test locally?

Reply:
Could we make it not specific to Mac OS? KITE has setup scripts for Linux and Windows too.

Reply:
We should, but I cannot write those docs.